docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -213,14 +212,8 @@ Once the release task has completed, it may be ideal to manually test the site o
 by running it locally (e.g. `http://localhost/release-3.3.4`) to do some smoke-testing
 and make sure that there are no obvious issues missed.
 
-Since `cow` will only run the unit test suite, you'll need to check
-the build status of Behat end-to-end tests manually on travis-ci.org.
-Check the badges on the various modules available on [github.com/silverstripe](http://github.com/silverstripe).
-
 It's also ideal to eyeball the git changes generated by the release tool, making sure
-that no translation strings were unintentionally lost, no malicious changes were
```

Comment:
I don't see how we can expect someone performing a release to know if there have been malicious changes to translation strings unless they speak 100+ languages

Reply:
Fair call. I think maybe this was more for obvious things like defacing (in English) or malicious HTML

Reply:
Yeah, I wrote this originally, and the intent was to look for malicious HTML. We shouldn't really allow HTML in translation strings at all, but that's much tougher to fix. I vote for keeping this line in there, we've had no end of trouble with translations, so some form of "translation sanity checking" is required.